### ISSUE TITLE
"Shared WOWEXEC" window stays open, among other issues

### ISSUE THREAD

**Issue by mrjoey98** [leecher1337/ntvdmx64#103]
Title explains everything. Upon launching a graphical program (I'm using Microsoft Musical Instruments for this example), this shows up via the taskbar and I can't close or hide it:
![Annotation 2020-09-02 234549](https://user-images.githubusercontent.com/49084307/92070297-9ed7f400-ed79-11ea-8b6f-9fb2151c2dac.png)
Also an issue with Microsoft Musical Instruments: Clicking any of the buttons (i.e. "Random," "Contents," etc) causes the program to hang and closing it throws me this error:
![Annotation 2020-09-02 234513](https://user-images.githubusercontent.com/49084307/92070496-2faecf80-ed7a-11ea-840b-0e0ad7e410bd.png)
And then this one:
![Annotation 2020-09-02 235210](https://user-images.githubusercontent.com/49084307/92070526-448b6300-ed7a-11ea-8d1d-96f16c8e39d1.png)
Another issue I noticed is that Voyetra's Digital Orchestrator Pro simply refuses to run, even with all the necessary DLL files loaded, yielding this error:
![Annotation 2020-09-02 234513](https://user-images.githubusercontent.com/49084307/92070451-068e3f00-ed7a-11ea-899f-646c712c06a2.png)
Lastly, there are a few programs that just don't run, citing that "Windows can't find the specified program," like this:
![Annotation 2020-09-02 234949](https://user-images.githubusercontent.com/49084307/92070540-54a34280-ed7a-11ea-8c6c-cb90f816a997.png)
By any chance are issues like these going to be fixed in a later release, or is there anything I can do now to mitigate them?
To my knowledge, I'm running the most recent version of NTVDMx64 (though, of course, I could be wrong) and I am using Windows 10 version 1909 (64-bit).

**Comment by leecher1337** [comment]
Thank you for testing. I have to say that Win16 support is still **very** experimental and in state of pure proof-of-concept (and unfortunately hard to debug). As I'm currently the only person working on this project, don't expect a solution anytime soon. It is still recommended to use [winevdm](https://github.com/otya128/winevdm) for Win16 support in production, as it is being actively developed, well supported and the NTVDMx64 loader detects its presence and hands over to winevdm in case it is installed, so you can use NTVDMx64 for DOS and winevdm for Win16 and have the best of both worlds.

**Comment by kristibektashi** [comment]
I discovered another related bug. UAC breaks NTVDMx64 support. If I try to run any 16-bit program on a 64-bit machine with NTVDMx64 it shows the above 'Windows cannot find filepath\filename.exe" error. Also, while shared wowexec window is open sometimes I cannot launch another win16 app before closing the shared wowexec window.

This has been tested using the latest columbia.edu version (october 2020)
